lib/testing/src/lib/core/pages/form/widgets/attach-file-widget.page.ts

```diff
@@ -122,6 +122,15 @@ export class AttachFileWidgetPage {
         return this.removeFileOptionButton.isEnabled();
     }
 
+    async isRemoveFileOptionDisplayed(): Promise<boolean> {
+        try {
+            await BrowserVisibility.waitUntilElementIsVisible(this.removeFileOptionButton);
```

Comment:
can you make removeFileOptionButton a testelement has already that method inside

Reply:
I've changed the `removeFileOptionButton` locator to TestElement